lntest/rpc/lnd.go

```diff
@@ -292,6 +292,24 @@ func (h *HarnessRPC) AddInvoice(req *lnrpc.Invoice) *lnrpc.AddInvoiceResponse {
 	return invoice
 }
 
+// AddAliases adds a list of aliases to the node's alias map.
+func (h *HarnessRPC) AddAliases(req *lnrpc.AddAliasesRequest) {
+	ctxt, cancel := context.WithTimeout(h.runCtx, DefaultTimeout)
+	defer cancel()
+
+	_, err := h.LN.AddAliases(ctxt, req)
+	h.NoError(err, "AddAliases")
+}
+
+// DeleteAliases deleted a set of alias mappings.
+func (h *HarnessRPC) DeleteAliases(req *lnrpc.DeleteAliasesRequest) {
```

Comment:
IMO we can keep these method names as they are and just move them to `lntest/rpc/router.go` and change the call to `h.Router.X...`.

Reply:
oh didn't see this comment and I went ahead and named them as they are in the RPC

why not keep them consistent? this way someone looking at the itest as reference can directly relate the call to the existing rpc method

Reply:
Was just to save you some time on the rename. But if you've matched everything up naming wise, even better.